bench/workgen/workgen.cxx

```diff
@@ -2324,6 +2583,24 @@ WorkloadOptions::WorkloadOptions()
       "timestamp forward");
     _options.add_bool("max_idle_table_cycle_fatal", max_idle_table_cycle_fatal,
       "print warning (false) or abort (true) of max_idle_table_cycle failure");
+    _options.add_int("dynamic_create_period", dynamic_create_period,
+      "if greater than 0, creates tables every this many seconds. The number of tables created "
+      "is specified by the dynamic_create_count setting");
+    _options.add_int("dynamic_create_count", dynamic_create_count,
+      "create this many number of tables every dynamic_create_period seconds");
```

Comment:
`number of tables to create each time interval`

Reply:
Done.